python/ak4_cff.py

```diff
@@ -0,0 +1,114 @@
+import FWCore.ParameterSet.Config as cms
+from  PhysicsTools.NanoAOD.common_cff import *
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+
+# ---------------------------------------------------------
+
+
+def setupCustomizedAK4(process, runOnMC=False, path=None):
+
+    # Mu subtraction
+    process.MuSubProducer = cms.EDProducer('MuSubProducer',
+        src=cms.InputTag('slimmedMuons'),
+        pfcs=cms.InputTag('packedPFCandidates'),
+        vtxs=cms.InputTag('offlineSlimmedPrimaryVertices'),
+        ptmin=cms.double(55.0)
+    )
+
+
+    bTagDiscriminators = [
+        'pfCombinedInclusiveSecondaryVertexV2BJetTags',
+        'pfDeepCSVJetTags:probb',
+        'pfDeepCSVJetTags:probbb'
+    ]
+    JETCorrLevels = ['L1FastJet', 'L2Relative', 'L3Absolute', 'L2L3Residual']
+
+    from PhysicsTools.NanoHRT.jetToolbox_cff import jetToolbox
+    jetToolbox(process, 'ak4', 'dummySeq', 'out', associateTask=False,
+               PUMethod='CHS', JETCorrPayload='AK4PFchs', JETCorrLevels=JETCorrLevels,
+	       postFix='MuSub',	
+	       newPFCollection=True, nameNewPFCollection="MuSubProducer",
+               Cut='pt > 30.0 && abs(rapidity()) < 2.4',
```

Comment:
I think this `Cut` is before applying the JEC, so maybe we need to leave some margin (e.g., pt>20 or 25)?

Reply:
done